src/app.controller.ts

```diff
@@ -4,20 +4,20 @@ import {
   HttpException,
   HttpStatus,
   Param,
+  Req,
   Res,
 } from '@nestjs/common';
 import { AppService } from './app.service';
 import { Public } from './auth/auth.decorator';
```

Comment:
**praise**: Good use of utility function `getFullUrl` to simplify URL construction.